src/components/Button/KeyBox/KeyBox.js

```diff
@@ -15,12 +18,17 @@ const StyledKeybox = styled(Col)`
   border-width: 1px;
   text-align: center;
   border-style: solid;
-  border-color: ${({ theme }) => theme.color.primary.default};
-  color: ${({ isActive, theme }) => isActive && theme.color.white.default};
+  border-color: ${({ theme }) =>
+    theme?.color?.primary?.default || typeformTheme?.color?.primary?.default};
```

Comment:
we were discussing how necessary is a border there. Why did you decide to keep it?